src/ardrive.ts

```diff
@@ -77,6 +116,57 @@ export class ArDriveAnonymous extends ArDriveType {
 		const driveEntity = await this.arFsDao.getPublicDrive(driveId);
 		return Promise.resolve(driveEntity);
 	}
+
+	async getPublicFolder(folderId: string): Promise<ArFSPublicFolder> {
+		const folder = await this.arFsDao.getPublicFolder(folderId);
+		return folder;
+	}
+
+	async getChildrenOfPublicFolder(folderId: FolderID): Promise<ArFSPublicFileOrFolderData[]> {
```

Comment:
I'm interesting in some team discussion around whether this logic ought to live in the ArDrive layer or the ArFS layer. My gut is that it should be the latter since this is rote data retrieval and organization based on the ArFS spec and not really business logic for the application layer. I'm open to other interpretations though.

Reply:
I think it makes sense to move those functions to ArFSDAO. Those do no more than fetching and sorting data.

Reply:
Applied at commit e7053907e8a0524d73e29ebae3c84b48c3442e9c